openage/opus/todo

```diff
@@ -0,0 +1,31 @@
+# I'm not sure if lines ending in '?' need to be done.
+'TODO's/'asdf's in source
+-> granulepos at EOS. opusenc from opus_tools does 'something':
+https://git.xiph.org/?p=opus-tools.git;a=blob;f=src/opusenc.c;h=129dcdb5dba2fa7caefb3b948d67b351e1755c0f;hb=5aecfedfb1290d4272e53cf12405cbc5b68f0ffe#l993
+
+create better ogg stream id?
+
+don't encode additional frame in main loop, if nb_samples == 0 (inputsize is multiple of frame_sz)
+
+Codestyle?
+exception handling?
+
+thread safe?
+
+register pyx/pxd files in the buildsystem.
+
+move openage/opus to openage/convert/opus?
+
+remove debug messages from opusenc.py?
+
+remove unused error constants in opus.pxd ?
+
+limit bandwidth for inputsignals with low samplerate ? check, if this is done automatical.
+
+copy metadata from Inputfile?
+
+see, why the upsampler is so slow.
```

Comment:
I read the opusenc.cpp and declared the variable 'ch' as with cdef int. for a 6 minute song the resampling now takes only 3 seconds instead of 30. total encoding time is now 15 seconds instead of 42.

the cython generated code looks pretty horrible to me. I hope the compiler optimizes most of. Where are the compilersettings for cython generated files?